rllib/evaluation/sample_batch_builder.py

```diff
@@ -29,6 +29,8 @@ def to_float_array(v: List[Any]) -> np.ndarray:
 class SampleBatchBuilder:
     """Util to build a SampleBatch incrementally.
 
+    TODO(sven): remove this once we switch to trajectory view API.
```

Comment:
Put TODOs as normal python comments, not in the pydoc.

Reply:
done